VideoAPI/Video.API/Validations/AddTaskRequestValidation.cs

```diff
@@ -0,0 +1,15 @@
+using FluentValidation;
+using VideoApi.Contract.Requests;
+
+namespace Video.API.Validations
+{
+    public class AddTaskRequestValidation : AbstractValidator<AddTaskRequest>
+    {
+        public const string NoTaskTypeError = "Task type is required";
+
+        public AddTaskRequestValidation()
+        {
+            RuleFor(x => x.TaskType).IsInEnum().WithMessage(NoTaskTypeError);
```

Comment:
Would you validate the body is not empty either please?

Reply:
added validation for participant and body